dm/dm/master/server.go

```diff
@@ -2421,3 +2423,18 @@ func (s *Server) sharedLogic(ctx context.Context, req interface{}, respPointer i
 	*errPointer = terror.ErrMasterRequestIsNotForwardToLeader
 	return true
 }
+
+// checkWorkerStatus when op is stop-task, need check worker status.
```

Comment:
we also use this func when `check whether source relative worker has been removed by scheduler` so please adjust this comment